packager/signer/signer.go

```diff
@@ -138,13 +138,26 @@ func noRedirects(req *http.Request, via []*http.Request) error {
 	return http.ErrUseLastResponse
 }
 
-func New(certHandler certcache.CertHandler, key crypto.PrivateKey, urlSets []util.URLSet,
+func New(certHandler certcache.CertHandler, key crypto.PrivateKey, publicDir string, urlSets []util.URLSet,
 	rtvCache *rtv.RTVCache, shouldPackage func() error, overrideBaseURL *url.URL,
 	requireHeaders bool, forwardedRequestHeaders []string) (*Signer, error) {
+	var rt http.RoundTripper
+	if publicDir != "" {
+		t := &http.Transport{}
+		r := http.NewFileTransport(http.Dir(publicDir))
+		t.RegisterProtocol("http", r)
+		t.RegisterProtocol("https", r)
+		log.Printf("AMP source is \"%s\"", publicDir)
```

Comment:
Maybe s/AMP source is/AMP pages will be fetched from/ would be more understandable.

Reply:
Yeah much better, thanks!